src/components/profile/projects/projects.js

```diff
@@ -1,10 +1,37 @@
 import React from 'react'
 import { connect } from 'react-redux'
+import classes from './projects.module.css'
 
 const Projects = props => {
     return (
         <div className={props.wrapper}>
-            <p>{props.title}</p>
+            <p className={classes.title}>Projects</p>
+            <div className={classes.container}>
+            {props.projects.map((project,index)=>
+            
+            
+                <div key={index} className={classes.card}>
+<div className={classes.head}>
+{project.title}
+</div>
+<div className={classes.body}>
+{project.description}
+
+<a href="#" className={classes.link}> More...</a>
+</div>
+
+
```

Comment:
Recommendation: adjust code lines and block, remove unnecessary spaces to keep the file concise.

Reply:
Also for the branch name it is recommended that you name your branch what you are working on i.e if you are adding skills name the branch feature/skill and if it is an issue that you are resolving name it issue/<issue number>